tests/bson.cpp

```diff
@@ -40,7 +38,7 @@ static inline std::string remove_whitespace(const char* c) {
  ======== BSON CORPUS ========
  */
 template <typename T>
-using CorpusCheck = std::function<bool(T)>;
```

Comment:
`std::function` can be brutal to compile times, especially if you are always constructing it from unique lambdas. Switching to a simple function pointer shaves about a second (10%) off my rebuild times.